### ISSUE TITLE
Request: Shortcut keys to change ink type

### ISSUE THREAD

**Issue by KasumiArai** [aseprite/aseprite#787]
With the new shading ink, I might want to more quickly change between it and a more "regular" ink type. It may even be nice to have "quick" ink selection like the quick tools.

Say my current ink is Alpha Compositing. If shading is "quick" bound to C, while I hold C the current tool will use the shading ink. Upon releasing it, the current ink will change back to Alpha Compositing.

**Comment by dacap** [comment]
Actually there are several things that would be great to automatically configured through a keystroke. For example, brushes can be changed with Alt+1, Alt+2, etc. but I think that we should be able to bind one of these shortcuts (e.g. Alt+1) into a specific set of options.

For example:
* Alt+1 change brush size=1px, ink=simple ink
* Alt+2 change ink=shading, and selected color range  [3,4,5] indexes.
etc.

These Alt+1,2,3,4... shortcuts should be configurable using a special UI with options like:
* Brush: Don't change / Use this: [ OPTIONS ]
* FG color: Don't change / Use this: [ FG ]
* BG color: Don't change / Use this: [ BG ]
* Ink mode, opacity, etc.

**Comment by dacap** [comment]
This might be related to #475, we could assign "macros" to keys, so steps would be something like:
* Record macro
* Select an ink, brush size, or other things/tasks (e.g. switching a specific layer visibility, executing a command, etc.)
* Stop the macro-recording
* Assign a specific key to that macro

**Comment by dacap** [comment]
Things missing to close this issue:

1. default shortcuts (we have already the commands that can be customized by the user)
2. "quick peek" modifier to change the ink type temporally when the key/modifier is pressed

I'm not sure about default shortcuts to change the ink type. The preferred way to switch between brushes/ink types will be `Alt+1`, `Alt+2`, etc. which should be implemented soon.

Also, the "quick peek" behavior is a little complex to be implemented for v1.1, I think it should be separated to other issue.

**Comment by dacap** [comment]
I'll move this for v1.2 as I think after #904 this feature will not be required anymore.
